src/integration/hubspotUpdatesController.ts

```diff
@@ -26,18 +28,29 @@ export class hubspotUpdatesController{
         // Hubspot does not include email in contact.created
         // Get new contact's email
         let params ='';
-        this.hubApiModel.getContact(contactObjectId, 'email')
+        this.hubApiModel.getContact(contactObjectId)
         .then(data =>{
             console.log("response");
             console.log(data);
+            const participant = data;
             params = `email:${encodeURIComponent(data.properties.email)}`;
+            console.log("PARAMS");
             console.log(params);
             // 1. Check if contact exists as user in SaaSquatch (match by email)
             this.saasApiModel.getUsers(params)
```

Comment:
does this correctly match on user email? I wan't able to get this working. That is, if there was a matching email in saasquatch it would still return 0 users.

Reply:
Not yet, that's a to do so I will add a comment. That said saasquatch automatically rejects the post if the email already exists as the ID so for demo it's not necessarily mandatory. Daniel and I are trying to get that functionality working now.